zerver/lib/test_helpers.py

```diff
@@ -465,9 +464,10 @@ def new_method(*args: Any, **kwargs: Any) -> Any:
             zerver.lib.upload.upload_backend = LocalUploadBackend()
     return new_method
 
-def create_s3_buckets(*bucket_names: Tuple[str]) -> List[Bucket]:
-    conn = S3Connection(settings.S3_KEY, settings.S3_SECRET_KEY)
-    buckets = [conn.create_bucket(name) for name in bucket_names]
+def create_s3_buckets(*bucket_names: Tuple[str]) -> List[Any]:
```

Comment:
I almost forgot to mention; I was also having trouble importing ‘Bucket’ via ‘boto3.Session.resource’ to use as a the return List type here for some reason. I’d get errors stating “...has no module Bucket”.

Reply:
(Which was confusing  because it’s the approach I used to annotate the buckets in within the tests for mypy.)

Reply:
https://github.com/python/typeshed/issues/2706 is relevant here.

Reply:
(Just changed this to be a `ServiceResource` type, along with the buckets in `test_upload`)